### DISCUSSION TITLE
Flutter - export .app to test on my iphone

### DISCUSSION THREAD

**Discussion by manatutorial** [codemagic-ci-cd/codemagic-docs#1461] [Q&A] [answered]
hello
i develop a flutter application on android and ios with vscode on windows 10 and i don't have a mac so i try to generate a .app on codemagic to test my app on my iphone but it seems there is a problem with my xcworkspace : 
my codemagic.yalm : 
workflows:
  simulator-native-ios:
    name: iOS simulator build
    max_build_duration: 120
    instance_type: mac_mini_m1
    environment:
      vars:
        XCODE_WORKSPACE: "Runner.xcworkspace"
        XCODE_SCHEME: "Runner"
      xcode: 13.3
      flutter: stable
      cocoapods: default
    scripts:
      - name: Install CocoaPods dependencies
        script: | 
          find . -name "Podfile" -execdir pod install \;
      - name: Build the .app
        script: | 
          xcodebuild build \
            -workspace "$XCODE_WORKSPACE" \
            -scheme "$XCODE_SCHEME" \
            -sdk iphonesimulator \
            -destination 'platform=iOS Simulator,name=iPhone 12 Pro,OS=15.4' \
            -configuration Debug \
            CODE_SIGN_IDENTITY="" \
            CODE_SIGNING_REQUIRED=NO \
            CODE_SIGNING_ALLOWED=NO 
    artifacts:
      - /tmp/xcodebuild_logs/*.log
      - $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.app
      - $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM

i got this error when codemagic try to build : 

2023-03-08 09:48:26.471 xcodebuild[1163:7423] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2023-03-08 09:48:26.471 xcodebuild[1163:7423] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
Command line invocation:
    /Applications/Xcode-13.3.app/Contents/Developer/usr/bin/xcodebuild build -workspace Runner.xcworkspace -scheme Runner -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 12 Pro,OS=15.4" -configuration Debug CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO

User defaults from command line:
    IDEPackageSupportUseBuiltinSCM = YES

Build settings from command line:
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO
    SDKROOT = iphonesimulator15.4

xcodebuild: error: 'Runner.xcworkspace' does not exist.

Build failed :|
Step 3 script `Build the .app` exited with status code 66

I ve verified and my xcworkspace is named Runner... Can you help me ?

Thank you

**Comment by himesh-cm** [comment, marked as answer]
@manatutorial The error says `Runner.xcworkspace`, can you check if you are running the command in the correct directory?
if you still need assistance, please share the build id, so we can look into the further

**Reply by manatutorial** [reply]
Thank you ! 
i change XCODE_WORKSPACE: "Runner.xcworkspace" to XCODE_WORKSPACE: "ios/Runner.xcworkspace"
and i don't have this error anymore
